### PR TITLE
[ENG-2391] Add review actions component test

### PR DESCRIPTION
**Pull request by futa-ikeda**
- Ticket: [ENG-2391]
- Feature flag: n/a

## Purpose
- add tests for the review-action component

## Summary of Changes
- move tests for review-action-list into registries directory
- add test file for review-action component

## Side Effects
NA

## QA Notes
NA

[ENG-2391]: https://openscience.atlassian.net/browse/ENG-2391

### CODE REVIEW

File: tests/integration/components/registries/review-actions-list/review-action/component-test.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    test('Embargo accept action', async function(assert) {
+        const reviewAction = this.get('reviewAction') as ReviewActionModel;
+        const embargoEndDate = faker.date.future(1, new Date(2099, 0, 0));
+        reviewAction.setProperties({
+            actionTrigger: ReviewActionTrigger.AcceptSubmission,
+            toState: RegistrationReviewStates.Embargo,
+        });
+        const dateString = formattedTimeSince(reviewAction.dateModified);
+        const embargoEndDateString = this.owner.lookup('service:intl').formatDate(embargoEndDate);
```
Comment (fabmiz) [inline tests/integration/components/registries/review-actions-list/review-action/component-test.ts:60]: you get `intl` service on the `this` context from `ember-intl/test-support` `(this: TestContext)` through `setupIntl(hooks)`
```suggestion
        const embargoEndDateString = this.intl.formatDate(embargoEndDate);
```

**Review by fabmiz — COMMENTED** [review]
Looks good overall. Are there any reasons you didn't cover actions log for `RejectWithdrawal`, `AcceptWithdrawal`, `RejectSubmission`?

File: tests/integration/components/registries/review-actions-list/review-action/component-test.ts (hunk 1 above)
Comment (futa-ikeda) [reply tests/integration/components/registries/review-actions-list/review-action/component-test.ts:60]: Ahh, I was having trouble getting my IDE to find `this.intl` , but I wasn't binding `this: TestContext` to the test...

**Comment by fabmiz** [comment]
@futa-ikeda could you rebase pls?

**Review by fabmiz — APPROVED** [review]
🥇 🏹

**Review by adlius — APPROVED** [review]

**Pull request merged**